tests/finance/test_inflate_currency.py

```diff
@@ -0,0 +1,137 @@
+from datetime import date, datetime
+
+import pytest
+import requests
+
+from janitor.finance import _inflate_currency, inflate_currency
+
+
+@pytest.mark.finance
+def test_make_currency_inflator_api_request():
```

Comment:
Something came to mind on how to improve tests that require internet connectivity. Maybe we can do "skip if cannot ping some generic, always-available server"?

Here's a suggestion.

```suggestion
@pytest.mark.skipif(...)  # some boolean that indicates whether we are able to ping internet or not?
def test_make_currency_inflator_api_request():
```

It's ok if you don't have an answer to this one - feel free to resolve it and we can move on without changing the test.

Reply:
I'm not sure off-hand how to handle this.  I'm going to resolve for now, and we can always add in such functionality later if/when we figure it out.